### ISSUE TITLE
Docstring text hover not properly

### ISSUE THREAD

**Issue by HydrogenSulfate** [microsoft/vscode#184128]
Does this issue occur when all extensions are disabled?: Yes

版本: 1.78.2 (user setup)
提交: b3e4e68a0bc097f0ae7907b217c1119af9e03435
日期: 2023-05-10T14:39:26.248Z
Electron: 22.5.2
Node.js: 16.17.1
OS: Windows_NT x64 10.0.19045

Steps to Reproduce:

I find that vscode hover for docstring can not properly process a long text without newline, as it will affect left-alignment of arguments below.

with newline after word "learning rate" for first argument `learning_rate`
![image](https://github.com/microsoft/vscode/assets/23737287/1a03fbe7-8bc2-4544-b33b-808332ddba56)

without newline after word "learning rate" for first argument `learning_rate`\
![image](https://github.com/microsoft/vscode/assets/23737287/9451d308-4099-4abf-b63d-b634b3df99e2)

Is there any one can help to solve this? 

demo code:
``` py
class SGD:
    """Stochastic Gradient Descent.

    Args:
        learning_rate (Union[float, optim.lr.LRScheduler], optional): The learning rate
            used to update parameter(s). Defaults to 0.001.
        weight_decay (Optional[Union[float, regularizer.L1Decay, regularizer.L2Decay]]):
            Regularization strategy. Defaults to None.
        grad_clip (Optional[Union[nn.ClipGradByNorm, nn.ClipGradByValue, nn.ClipGradByGlobalNorm]]):
            Gradient cliping strategy. Defaults to None.

    Examples:
        >>> import ppsci
        >>> model = ppsci.arch.MLP(("x",), ("u",), 5, 20)
        >>> opt = ppsci.optimizer.SGD(1e-3)((model,))
    """

    def __init__(
        self,
        learning_rate: Union[float, optim.lr.LRScheduler] = 0.001,
        weight_decay: Optional[
            Union[float, regularizer.L1Decay, regularizer.L2Decay]
        ] = None,
        grad_clip: Optional[
            Union[nn.ClipGradByNorm, nn.ClipGradByValue, nn.ClipGradByGlobalNorm]
        ] = None,
    ):
        self.learning_rate = learning_rate
        self.weight_decay = weight_decay
        self.grad_clip = grad_clip
```

**Comment by VSCodeTriageBot** [comment]
It looks like this is caused by the Python extension. Please file the issue to the [Python extension repository](https://github.com/microsoft/vscode-python). Make sure to check their issue reporting template and provide them relevant information such as the extension version you're using. See also our [issue reporting guidelines](https://aka.ms/vscodeissuereporting) for more information.

Happy Coding!
